sormas-api/src/main/java/de/symeda/sormas/api/sample/SampleExportDto.java

```diff
@@ -787,6 +715,48 @@ public ContactJurisdictionDto getAssociatedContactJurisdiction() {
 		return associatedContactJurisdiction;
 	}
 
+	public SamplePathogenTest getPathogenTest1() {
+		return pathogenTest1;
+	}
+
+	public void setPathogenTest1(SamplePathogenTest pathogenTest1) {
+		this.pathogenTest1 = pathogenTest1;
+	}
+
+	public SamplePathogenTest getPathogenTest2() {
+		return pathogenTest2;
+	}
+
+	public void setPathogenTest2(SamplePathogenTest pathogenTest2) {
+		this.pathogenTest2 = pathogenTest2;
+	}
+
+	public SamplePathogenTest getPathogenTest3() {
+		return pathogenTest3;
+	}
+
+	public void setPathogenTest3(SamplePathogenTest pathogenTest3) {
+		this.pathogenTest3 = pathogenTest3;
+	}
+
+	public List<SamplePathogenTest> getOtherPathogenTests() {
+		return otherPathogenTests;
+	}
+
+	public static class Material {
```

Comment:
It Is necessary because the `sampleMaterialDetails` should be pseudonymized but it looks like I forgot to add the annotation :)

I will rename these classes